install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -65,8 +65,9 @@ type WorkspaceConfig struct {
 }
 
 type WebAppConfig struct {
-	PublicAPI *PublicAPIConfig `json:"publicApi,omitempty"`
-	Server    *ServerConfig    `json:"server,omitempty"`
+	PublicAPI      *PublicAPIConfig `json:"publicApi,omitempty"`
+	Server         *ServerConfig    `json:"server,omitempty"`
+	UsePodAffinity bool             `json:"usePodAffinity"`
```

Comment:
nit: Should read **anti**: `UsePodAntiAffinity bool             `json:"usePodAntiAffinity"`